cmd/chimera/server.go

```diff
@@ -32,14 +32,17 @@ func newServeCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().String("addr", "localhost:9876", "Address on which server should be listening.")
-
+	cmd.Flags().Int("port", 9876, "Port on which server should be listening.")
+	cmd.Flags().String("address", "", "External address of Chimera service.")
 	cmd.Flags().String("apps-config", "apps-config.json", "Path to the file containing OAuth apps configuration.")
 	cmd.Flags().String("cache-driver", "inMemory", "Cache driver to be used by application, one of: inMemory, redis.")
 	cmd.Flags().String("cache-expiration", "10m", "Cache expiry time.")
 	cmd.Flags().String("redis-address", "localhost:6379", "Redis server address (required if cache-driver is redis).")
 	cmd.Flags().String("redis-password", "", "Redis server password.")
 	cmd.Flags().Int("redis-database", 0, "Redis database.")
+	cmd.Flags().String("confirmation-template-path", "html/confirm-auth-form.html", "HTML template for authorization confirmation form.")
+	cmd.Flags().String("cancel-page-path", "html/cancel-auth.html", "HTML page displayed when canceling authorization.")
+	cmd.Flags().String("csrf-secret", "dSgVkYp3s6v9y$B?E(H+MbQeThWmZq4t", "32 byte secret used for generating CSRF tokens.")
```

Comment:
This is a little confusing for multiple reasons:

 - A secret like this should never have a default value. If it's necessary to specify a secret for a secure deployment, not specifying one should cause a fatal error.
 - The secret isn't actually used for generating CSRF tokens so the help text is inaccurate. It's used by the CSRF library, but only for keying HMACs in `gorilla/securecookie`
 - Authenticating the CSRF cookie isn't actually necessary in any scenario; the CSRF library is just silly. It's fine otherwise, there's just unnecessary complexity and secrets management for no gain.
 
 I'd suggest getting rid of the whole `csrf-secret` flag and using an obviously non-secret hardcoded value to initialize the CSRF library -- together with a comment explaining what the library uses the "secret" for.

Reply:
Thanks @jupenur, I removed the flag and hardcoded secret as per your suggestion.